eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -58,9 +58,8 @@ parameters:
   - name: TestProxy
     type: boolean
     default: false
-  - name: ToxTestEnv
```

Comment:
Any chance we can keep `ToxTestEnv` so the optional environment can still be passed in?

Reply:
> Any chance we can keep `ToxTestEnv` so the optional environment can still be passed in?

Yeah I'll add it back in. Thanks for the catch!